com.ktechpit.colorwall.yaml

```diff
@@ -0,0 +1,90 @@
+app-id: com.ktechpit.colorwall
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: colorwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --device=dri
+  - --env=PACKAGE_TYPE=flatpak
+  - --talk-name=org.freedesktop.FileManager1
+  - --filesystem=xdg-run/dconf
+  - --filesystem=xdg-run/gvfsd
+  - --filesystem=~/.config/dconf:ro
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+
+modules:
+  - name: lzo
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    cleanup:
+      - /include
+      - /share/doc
+      - '*.la'
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: http://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+
+
+  - name: squashfs-tools
+    buildsystem: simple
+    build-commands:
+      - XZ_SUPPORT=1 LZO_SUPPORT=1 make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=${FLATPAK_DEST}/bin
+    cleanup:
+      - '*'
+    sources:
+    - type: git
+      url: https://github.com/plougher/squashfs-tools.git
+      tag: '4.5'
+      commit: '0496d7c3de3e09da37ba492081c86159806ebb07'
+
+
+  - name: libgumbo
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/google/gumbo-parser/archive/v0.10.1.tar.gz
+        sha256: 28463053d44a5dfbc4b77bcf49c8cee119338ffa636cc17fc3378421d714efad
+    cleanup:
+      - /lib/pkgconfig
+      - /include
+      - '*.a'
+      - '*.la'
+
+  - name: colorwall
+    buildsystem: simple
+    build-commands:
+      - unsquashfs -no-progress tPiGTcpRYwWhmLFqKMefGzuC6isfwbci_6.snap
+      - install -Dm755 squashfs-root/usr/bin/colorwall ${FLATPAK_DEST}/bin/colorwall
+      - install -d ${FLATPAK_DEST}/lib
+      - mv squashfs-root/usr/lib/* ${FLATPAK_DEST}/lib/
+      - install -Dm644 squashfs-root/meta/gui/icon.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/com.ktechpit.colorwall.png
```

Comment:
why is there icon files in this repository if you extract it here....

Reply:
I removed the upstream one. Using the one maintained in this repository.

Reply:
NO. I really meant it the way I meant it.

Reply:
Like get rid of icons from here completely? Sorry for misunderstanding.

Reply:
yes like that.